### DISCUSSION TITLE
Lab Assignment #3 👩‍💻

### DISCUSSION THREAD

**Discussion by albertkun** [albertkun/24SU-ASIAAM-191A#42] [Labs]
## Due 7/17 before class 

Please use this thread to post your [lab assignment's #4](https://albertkun.github.io/24SU-ASIAAM-191A/assignments/week3/lab_assignment/) link, which is the GitHub pages URL. [Click here for a reminder on how to get it](https://albertkun.github.io/24SU-ASIAAM-191A/labs/week1/3/#using-github-pages).

### For example

https://albertkun.github.io/24su-asiaam-191a-sample/week3/index.html

**Comment by sjeyakum** [comment]
https://sjeyakum.github.io/24su-aas191a-labs/week3/index.html

**Comment by stinest** [comment]
https://stinest.github.io/24su-191a-labs/week3/index.html

**Comment by kianaptran** [comment]
https://kianaptran.github.io/24su-asiaam-191a-labs/week3/

**Comment by jasminereyz** [comment]
https://jasminereyz.github.io/AA191-Labs/week3/index.html

**Reply by pwinnie2026** [reply]
https://pwinnie2026.github.io/24su-asiaam-191a-labs/Week3Extra/index.html

**Comment by zhamic7** [comment]
https://zhamic7.github.io/aa191a-labs/week3/index.html

**Comment by nesslee123** [comment]
https://nesslee123.github.io/Assignments/week3/index.html

**Comment by Keiya-Wada** [comment]
https://keiya-wada.github.io/asiaam-labs/week3/

**Comment by kchhoeu** [comment]
https://kchhoeu.github.io/24su-asiaam191a-labs/week3/

**Comment by chrissanrow** [comment]
https://chrissanrow.github.io/asiaam191a-su24/week3/

**Comment by kimada62** [comment]
https://kimada62.github.io/24su-asiaam-191a-labs/week3/

**Comment by giaari15** [comment]
https://giaari15.github.io/aas191-labs/week_3/

**Comment by NellieNguyen149** [comment]
https://nellienguyen149.github.io/24su-asianam-191a-labs/week3/index.html

**Comment by caitlynn-hauw** [comment]
https://caitlynn-hauw.github.io/aa191-assignments/week3/

In all fairness, I gave up. I did the lab, but it's not very pretty!
